### PR TITLE
docs: img add default alt and format table

### PR DESCRIPTION
**Pull request by zhouyou12**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design-web3/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

## 📝 Git Commit Message Convention

> This is adapted from [Angular's commit convention](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular).

### CODE REVIEW

File: docs/course/contract-compile-deploy.md [hunk 1]
```diff
@@ -47,11 +47,11 @@ Click the drop-down box of `ENVIRONMENT`, you can see many options to choose fro
 
 Then, `MetaMask` will pop up a window and prompt to connect it to `Remix`. After connecting `MetaMask` to `Remix`, the side panel will be updated to display the connected network and account. Deploying to the testnet also requires testnet tokens. You can find the corresponding testnet token faucet on the network for recharging.
 
-<img src='./img/changeNode.png' width="33%">
+<img src='./img/changeNode.png' width="33%" alt="">
```
Comment (kiner-tang) [inline docs/course/contract-compile-deploy.md:50]: 既然加了 alt 属性，就不要弄空字符串吧，根据图片内容写个概要放在 alt 里面

**Review by jeasonstudio — CHANGES_REQUESTED** [review]
PR title 需要改为符合 commit lint 格式的英文标题，squash merge 后风格统一

File: docs/course/contract-compile-deploy.md [hunk 2]
```diff
@@ -48,11 +48,11 @@ Click the drop-down box of `ENVIRONMENT`, you can see many options to choose fro
...
```
Comment (thinkasany) [inline docs/course/contract-compile-deploy.md:51]:
```suggestion
<img src='./img/changeNode.png' width="33%" alt="choose Injected Provider - MetaMask" />
```

File: docs/course/contract-compile-deploy.md [hunk 3]
```diff
@@ -48,11 +48,11 @@ Click the drop-down box of `ENVIRONMENT`, you can see many options to choose fro
...
 
-<img src='./img/connect1.png' width="33%">
+<img src='./img/connect1.png' width="33%" alt="">
```
Comment (thinkasany) [inline docs/course/contract-compile-deploy.md:53]:
```suggestion
<img src='./img/connect1.png' width="33%" alt="connecting MetaMask to Remix" />
```

File: docs/course/contract-compile-deploy.md [hunk 4]
```diff
@@ -48,11 +48,11 @@ Click the drop-down box of `ENVIRONMENT`, you can see many options to choose fro
...
 
-<img src='./img/connect2.png' width="33%">
+<img src='./img/connect2.png' width="33%" alt="">
```
Comment (thinkasany) [inline docs/course/contract-compile-deploy.md:55]:
```suggestion
<img src='./img/connect2.png' width="33%" alt="display the connected network and account" />
```

File: docs/course/contract-compile-deploy.zh-CN.md [hunk 5]
```diff
@@ -47,9 +47,9 @@ order: 3
 
 随后，`MetaMask` 将跳出弹窗并提示将其连接至 `Remix`。将`MetaMask` 连接至 `Remix` 后，侧面板将更新显示所连接的网络和账户。部署到测试网也需要测试网的代币，你可以在网络上查找对应的测试网代币水龙头充值使用。
 
-<img src='./img/changeNode.png' width="33%">
-<img src='./img/connect1.png' width="33%">
-<img src='./img/connect2.png' width="33%">
+<img src='./img/changeNode.png' width="33%" alt="">
+<img src='./img/connect1.png' width="33%" alt="">
+<img src='./img/connect2.png' width="33%" alt="">
```
Comment (thinkasany) [inline docs/course/contract-compile-deploy.zh-CN.md:52]:
```suggestion
<img src='./img/changeNode.png' width="33%" alt="选择 Injected Provider - MetaMask" />
<img src='./img/connect1.png' width="33%" alt="将 MetaMask 连接到 Remix" />
<img src='./img/connect2.png' width="33%" alt="显示连接的网络和账户" />
```

**Review by thinkasany — APPROVED** [review]

**Review by jeasonstudio — APPROVED** [review]

**Comment by thinkasany** [comment]
@zhouyou12 Thanks for your contribution!

**Pull request merged**